contracts/BRTToken.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract BRTToken is ERC20, Ownable {
+    uint constant _supply = 10000;
+
+    constructor() ERC20("BRTToken", "BRT") {
+        _mint(msg.sender, _supply);
```

Comment:
ile tokenów mintujesz deployerowi?

Reply:
dokładnie. domyślnie (można sobie nadpisać) `decimals() = 18` w ERC20 od openzeppelin, czyli mintujesz tutaj `1e5 / 1e18` swojego tokena.
Nota bene, w solidity bardzo dobrze z powyższego powodu sprawdza się zapis `xe18` np. 123e18 = 123 tokeny. Wszystkie operacje w smart contractach robisz na liczbach dodatnich, stałoprzecinkowych. Trzeba na to uważać, bo łatwo zgubić precyzję